sklearn/metrics/regression.py

```diff
@@ -42,17 +47,29 @@ def _check_reg_targets(y_true, y_pred):
 
     y_pred : array-like,
 
+    multioutput : array-like or string in ['raw_scores', uniform_average',
+        'variance_weighted'] or None
+
     Returns
     -------
     type_true : one of {'continuous', continuous-multioutput'}
         The type of the true target data, as output by
-        ``utils.multiclass.type_of_target``
+        'utils.multiclass.type_of_target'
 
-    y_true : array-like of shape = [n_samples, n_outputs]
+    y_true : array-like of shape = (n_samples, n_outputs)
         Ground truth (correct) target values.
 
-    y_pred : array-like of shape = [n_samples, n_outputs]
+    y_pred : array-like of shape = (n_samples, n_outputs)
         Estimated target values.
+
+    multioutput : array-like of shape = (n_outputs) or string in ['raw_scores',
+        uniform_average', 'variance_weighted'] or None
+        Custom output weights is multioutput is array-like. 'uniform_average'
+        and 'variance_weighted' indicate specific weight vectors.
+        'raw_scores' indicates no agglomeration of scores across targets:
+        scores are returned separately per target.
+        None is left due to backward compatibility of r2_score()
```

Comment:
I think I will just remove description of multioutput keywords from this docstring as in this function doesn't use them, just checks the validity.